src/common/components/outline/Outline.tsx

```diff
@@ -1,29 +1,38 @@
 import React from "react";
-import { Accordion, Box, Flex } from "@chakra-ui/react";
+import { Accordion, Box, Flex, Link } from "@chakra-ui/react";
 import { GButton } from "common/components/GButton";
 
 import { UIOutline } from "domain/common/UIOutline";
 import { UISection } from "domain/common/UISection";
 import { Section } from "./Section";
 import { domainUpdateAndCallback } from "common/utils";
+import { MdEdit } from "react-icons/md";
 
 type Props = {
   uiOutline: UIOutline | null;
   checkingEnabled?: boolean;
   taskSelectionEnabled?: boolean;
   onUpdateOutlineCallback: (uiOutline: UIOutline) => void;
+  allowOutlineChanges?: boolean;
+  expandByDefault?: boolean;
 };
 
 export const Outline: React.FC<Props> = ({
   uiOutline,
   checkingEnabled = false,
   taskSelectionEnabled = false,
   onUpdateOutlineCallback,
+  allowOutlineChanges = false,
+  expandByDefault = false,
 }) => {
   if (!uiOutline) {
     return null;
   }
 
+  if (expandByDefault) {
```

Comment:
in this case we will never be able to compress an outline section, therefore we should rename this to alwaysExpand or disableCollapse or something more explicit, what do you think?

Reply:
Done